react/features/invite/components/InviteButton.web.js

```diff
@@ -0,0 +1,231 @@
+const DIAL_OUT_OPTION = 'dialout';
+const ADD_TO_CALL_OPTION = 'addtocall';
+
+/**
+ * The button that provides different invite options.
+ */
+class InviteButton extends Component {
+    /**
+     * {@code InviteButton}'s property types.
+     *
+     * @static
+     */
+    static propTypes = {
+        /**
+         * Indicates if the "Add to call" feature is available.
+         */
+        _isAddToCallAvailable: PropTypes.bool,
+
+        /**
+         * Indicates if the "Dial out" feature is available.
+         */
+        _isDialOutAvailable: PropTypes.bool,
+
+        /**
+         * The function opening the dialog.
+         */
+        openDialog: PropTypes.func,
+
+        /**
+         * Invoked to obtain translated strings.
+         */
+        t: PropTypes.func
+    };
+
+    /**
+     * Initializes a new {@code InviteButton} instance.
+     *
+     * @param {Object} props - The read-only properties with which the new
+     * instance is to be initialized.
+     */
+    constructor(props) {
+        super(props);
+
+        this._onInviteClick = this._onInviteClick.bind(this);
+        this._onInviteOptionSelected = this._onInviteOptionSelected.bind(this);
+        this._updateInviteItems = this._updateInviteItems.bind(this);
+
+        this._updateInviteItems(this.props);
+    }
+
+    /**
+     * Implements React's {@link Component#componentWillReceiveProps()}.
+     *
+     * @inheritdoc
+     * @param {Object} nextProps - The read-only props which this Component will
+     * receive.
+     * @returns {void}
+     */
+    componentWillReceiveProps(nextProps) {
+        if (this.props._isDialOutAvailable !== nextProps._isDialOutAvailable
+            || this.props._isAddToCallAvailable
+                !== nextProps._isAddToCallAvailable) {
+            this._updateInviteItems(nextProps);
+        }
+    }
+
+    /**
+     * Renders the content of this component.
+     *
+     * @returns {ReactElement}
+     */
+    render() {
+        const { t } = this.props;
+
+        const { VERTICAL_FILMSTRIP } = interfaceConfig;
+
+        return (
+            <div className = 'filmstrip__invite'>
+                <div className = 'invite-button-group'>
+                    <Button
+                        onClick = { this._onInviteClick }
+                        shouldFitContainer = { true }>
+                        { t('invite.invitePeople') }
+                    </Button>
+                    { this.props._isDialOutAvailable
+                        || this.props._isAddToCallAvailable
+                        ? <DropdownMenu
+                            items = { this.state.inviteOptions }
+                            onItemActivated = { this._onInviteOptionSelected }
+                            position = { VERTICAL_FILMSTRIP
+                                ? 'bottom right'
+                                : 'top right' }
+                            shouldFlip = { true }
+                            triggerType = 'button' />
```

Comment:
What do you think about the arrow in the dropdown in horizontal filmstrip mode? It currently points down but the dropdown opens above.

Reply:
Do you know any property of the AtlasKit dropdown that can fix this? I personally thought that I can't change this.

Reply:
In atlaskit using their default dropdown menu? No, I don't see a property. You can use a custom icon/trigger but it's probably not worthwhile. It's cold to say but it's an atlaskit issue because even in their storybook the arrow doesn't always point to where the dropdown opens.

Reply:
Yeah, I would leave that one to them and report it there.